README.md

```diff
@@ -0,0 +1,47 @@
+# Infoblox Go Client
+
+An Infoblox Client library for Go.
+
+This library is compatible with Go 1.2+
+
+- [Usage](#Usage)
```

Comment:
Usage and Example section are same, re-arrange the order like this 

1. prerequisites
2. Installation
3. Usage/Example